### DISCUSSION TITLE
L1 Nodes — Recommended hosting providers / Experiences

### DISCUSSION THREAD

**Discussion by vmatekole** [filecoin-project/saturn-node#290] [Q&A] [answered]
Hi,
Looking for anyone working on IPFS Saturn. We are looking to spin-up a L1 node but are looking for those that already have experience and perhaps suggestions for hosting providers.

**Comment by DiegoRBaquero** [comment, marked as answer]
Best place is the #filecoin-saturn channel in the Filecoin Slack. You are invited to join: https://filecoin.io/slack
